src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -190,6 +199,26 @@ private void mockRunQueryForConcepts(
     }
   }
 
+  private void mockRunQueryForGetConcepts(
+      CloudPlatform cloudPlatform,
+      SnapshotBuilderConcept concept,
+      Dataset dataset,
+      String domainId) {
+    CloudPlatformWrapper cloudPlatformWrapper = CloudPlatformWrapper.of(cloudPlatform);
+    if (cloudPlatformWrapper.isGcp()) {
```

Comment:
Seems like a good case for using `choose()`?

Reply:
yeah, the only thing is since these method don't return anything and choose takes a supplier, I need to add a return null statement. What do you think about that?

cloudPlatformWrapper.choose(
        () -> {
          when(datasetService.retrieveModel(dataset, TEST_USER)).thenReturn(makeDatasetModel());
          when(bigQueryDatasetPdao.<SnapshotBuilderConcept>runQuery(any(), any(), any()))
              .thenReturn(concepts);
          return null;
        },
        () -> {
          when(datasetService.getOrCreateExternalAzureDataSource(dataset, TEST_USER))
              .thenReturn("dataSource");
          when(azureSynapsePdao.<SnapshotBuilderConcept>runQuery(any(), any()))
              .thenReturn(concepts);
          return null;
        });